spec/intro.md

```diff
@@ -10,7 +10,7 @@
    1. [`message.abnf`](message.abnf)
 1. [Formatting](formatting.md)
 1. [Errors](errors.md)
-1. [Default Function Registry](registry.md)
+1. [Default Function Registry](functions/README.md)
 1. [`u:` Namespace](u-namespace.md)
```

Comment:
Should the `u:` namespace be in the functions directory? Or in its own subdirectory?